POC/ColorPicker/ColorPicker/MainWindow.xaml.cs

```diff
@@ -109,5 +145,22 @@ private void DeactivateColorSelectionMode()
             _transparentWindow.Hide();
             _updateTimer.Stop();
         }
+
+        private void CopyToClipboard(TextBox textBox)
```

Comment:
CopyToClipboardAndOpenPopup? Or separate in two functions.

Reply:
Copying to the clipboard is only the first line. I'll extract the pop-up opening to its own function and call the clipboard copying on its own line.